numpy/typing/tests/data/pass/array_constructors.py

```diff
@@ -114,6 +114,8 @@ def func(i: int, j: int, **kwargs: Any) -> SubClass:
 
 np.atleast_3d(C)
 
+np.atleast_nd(C)
```

Comment:
I don't know if I did the right thing here. Someone should check...

Reply:
This is the correct place (same as with https://github.com/numpy/numpy/pull/18386/files#r573619117). Don't forget about the `ndim` parameter though.

Reply:
@BvB93. I actually don't know what this is doing. Could you please spell out what I should do with `ndim` and `pos`?

Reply:
The typing tests (`reveal` specifically) are used for verifying that the type that the type that is expected to-be returned _is_ actually returned (with the help of mypy). The `pass` tests furthermore check that the code actually runs at runtime, in addition to passing the type checkers unscathered.

Reply:
Note that the `ndim` parameter is still missing in both the `pass` and `reveal` tests (this _will_ raise).

Reply:
Thanks. I did not realize that these were tests. Now that you cut through my mental fog, I think I got them fixed up.